src/openrct2/rct1.h

```diff
@@ -29,7 +29,12 @@
 
 #define RCT1_MAX_MAP_ELEMENTS       0xC000
 #define RCT1_MAX_SPRITES            5000
-#define RCT1_MAX_VEHICLES_PER_RIDE  12
+#define RCT1_MAX_STATIONS           4
```

Comment:
I already named it RCT12_MAX_STATIONS_PER_RIDE (as you said in https://github.com/OpenRCT2/OpenRCT2/pull/5119#discussion_r98221816), do you want me to rename it to RCT12_MAX_STATIONS instead?